e3sm_to_cmip/__main__.py

```diff
@@ -717,9 +720,40 @@ def _run_info_mode(self):  # noqa: C901
                             break
                     if not has_vars:
                         continue
-                    messages.append(msg)
 
-        if self.output_path is not None:
+                    # We test here against the input "freq", because atmos mon data satisfies BOTH
+                    # CMIP6_day.json AND CMIP6_mon.json, but we only want the latter in the "hand_msg" output.
+                    # The variables-check below is insufficient as vars "hass" and "rlut" have multiple freqs.
+
+                    if self.freq == "mon" and handler['table'] == "CMIP6_day.json":
+                        continue
+                    if self.freq == "day" and handler['table'] == "CMIP6_Amon.json":
+                        continue
+
+                    hand_msg = None
+                    stat_msg = None
```

Comment:
```suggestion
```

Probably don't need to set the default value of `stat_msg` to `None` if we don't want to print out `None` values.